pkg/api/validate/dynamic/dynamic.go

```diff
@@ -53,6 +53,7 @@ type Dynamic interface {
 	ValidateQuota(ctx context.Context, oc *api.OpenShiftCluster) error
 	ValidateDiskEncryptionSets(ctx context.Context, oc *api.OpenShiftCluster) error
 	ValidateEncryptionAtHost(ctx context.Context, oc *api.OpenShiftCluster) error
+	ValidateVmSku(ctx context.Context, location string, subscriptionID string, oc *api.OpenShiftCluster) error
```

Comment:
I believe acronyms in the middle of the function name should be all caps.

```suggestion
	ValidateVMSku(ctx context.Context, location string, subscriptionID string, oc *api.OpenShiftCluster) error
```